src/clinvar_streams/storage/database_sqlite/client.clj

```diff
@@ -2,41 +2,71 @@
   (:require [clojure.java.io :as io]
             [clojure.java.jdbc :as jdbc]
             [clojure.java.shell :refer :all]
-            [taoensso.timbre :as log])
+            [taoensso.timbre :as log]
+            [clinvar-streams.util :as util]
+            [clinvar-combiner.config :as config])
   (:import [java.io File]
            (java.sql PreparedStatement)
            (java.util Iterator)))
 
 (def db (atom {}))
 
-(def sql-root "src/clinvar_streams/storage/database_sqlite/sql")
+(def sql-resource-root "clinvar_streams/storage/database_sqlite/sql")
```

Comment:
config param vs hardcoding?

Reply:
I think hardcoding is okay since it's a static file packaged into the jar file. Though it is also relative to the current package of that source code, just adding `/sql` to the end, so maybe that's a more reliable way to do it, in case the package is ever moved.